### PR TITLE
migrate sonos binding

### PR DESCRIPTION
**Pull request by maggu2810**

### CODE REVIEW

**Comment by J-N-K** [comment]
I think you are missing the CODEOWNERS change. Otherwise LGTM.

**Comment by maggu2810** [comment]
Good point!

File: addons/binding/org.openhab.binding.sonos/OSGI-INF/.gitignore [hunk 1]
```diff
@@ -1 +0,0 @@
-/*.xml
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.sonos/OSGI-INF/.gitignore]: If I understood it correctly from the webui repo the OSGI-INF is/was still generated so the .gitignore was still needed for Eclipse, is that also the case here? Or does that not apply any more?

Comment (wborn) [reply addons/binding/org.openhab.binding.sonos/OSGI-INF/.gitignore]: I figured it out later that was due to the fact the Mavenized projects had to be deleted and reimported into Eclipse. After that Eclipse doesn't generate these files anymore. :smile:

Comment (maggu2810) [reply addons/binding/org.openhab.binding.sonos/OSGI-INF/.gitignore]: You figured out that I missed to do that step once.
I need to import the project and add the generated Eclipse IDE files again.
Will add another commit later.

**Pull request merged**